CODEOWNERS

```diff
@@ -31,6 +31,7 @@ homeassistant/components/asuswrt/* @kennedyshead
 homeassistant/components/auth/* @home-assistant/core
 homeassistant/components/automatic/* @armills
 homeassistant/components/automation/* @home-assistant/core
+homeassistant/components/avea/* @pattyland
```

Comment:
Also add this to .coveragerc as seen in the PR template.

Reply:
I was really tryning, I swear 🙈
Fixed in 6f8e1b758bb27d2f299b0c37268c3c9ea8e1436d